docs/UserGuide.md

```diff
@@ -13,151 +27,211 @@ PeopleSoft is a desktop app for **calculating the salary for shift-based contrac
+| `find`      | `find KEYWORD [MORE_KEYWORDS]…​`                                                 | `find Nicole Hardware`, `find Aircon`                                                                    |
+| `list`      | `list`                                                                           | NA                                                                                                       |
+| `clear`     | `clear`                                                                          | NA                                                                                                       |
+| `job`       | `job JOBID n/NAME r/RATE d/DURATION`                                             | `job 1 n/Fix HDB Lock r/40 d/1`                                                                          |
+| `joblist`   | `joblist`                                                                        | NA                                                                                                       |
+| `jobdelete` | `jobdelete JOBID`                                                                | `jobdelete 3`                                                                                            |
+| `mark`      | `mark JOBID`                                                                     | `mark 2`                                                                                                 |
+| `assign`    | `assign JOBID i/INDEX`                                                           | `assign 2 i/1`                                                                                           |
+
+--------------------------------------------------------------------------------------------------------------------
 
 ## Features
 
 <div markdown="block" class="alert alert-info">
 
 **:information_source: Notes about the command format:**<br>
 
-* Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
-  e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
+* Words in `UPPER_CASE` are the attributes to be filled in by you.<br>
+  e.g. in `add n/NAME`, the `NAME` attribute could be `add n/John Doe`.
+ 
+* Attributes can be in any order.<br>
+  e.g. if the command asks for `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also fine.
 
 * Items in square brackets are optional.<br>
-  e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
-
-* Items with `…`​ after them can be used multiple times including zero times.<br>
-  e.g. `[t/TAG]…​` can be used as ` ` (i.e. 0 times), `t/friend`, `t/friend t/family` etc.
+  e.g `n/NAME [t/TAG]` can be typed as `n/John Doe t/friend` or as `n/John Doe`.
 
-* Parameters can be in any order.<br>
-  e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
+* Items with `…​` after them can be used multiple times including zero times.<br>
```

Comment:
are we gonna use the funky unicode ellipse or just three periods?

Reply:
the funky one